### ISSUE TITLE
"ВОЛГОСНАБТЕХ" К032087

### ISSUE THREAD

**Issue by Shtukin** [ashustov/test#83]
Нужна оценка, в прошлом году этих дилеров оценивал Лёвкин по моему, а может и ты. 
Проблема в том, что не было обязательных полей, но дилер их добавил.
Нужно повнимательней посмотреть. 

Замечания которые были после первой оценки:
1. В конфигурации учет автомобилей ведется как текстовое поле в заказ-наряде. 
Его нужно переводить на справочник с привязкой минимально необходимого набора реквизитов. 
Некоторые вещи для этого у уже сделаны. Например справочник моделей и марки.
2. Доверенное лицо предполагаю брать из поля "Грузополучатель"
3. Не нашел где в заказ-наряде отражается мастер. 
4. В справочнике услуг нет артикула. вместо него можно код грузить. Насколько это критично для УАЗа? (для работ это не критично)
5. По классификации. Ниже мое видение. 
         Подтвердите правильность предположений или указать какие поля правильно брать за основу.
 - Вида ремонта у них всего два (ТО и ТР)-это поле сопоставляем со списком видов ремонта
 - в заказ-наряде есть галка "Гарантия завода-изготовителя"- это поле сопоставляем с типами оплат. Если стоит галка- Гарантия, если нет- обычный ремонт
 - цех для сопоставления берем из поля склад заказ-наряда

И вот последнее письмо от дилера с правками и самой базой:
высылаю вам базу (архив RAR-5 каталога базы 7.7, база крутится на W2003S, поэтому кодировка базы соответственно - 1251);
добавлен реквизит (индексный, текст, 30) Артикул в справочник Услуги;

решено (с Дмитрием) в Номенклатуре использовать "Артикул производителя";
на счет доверенного лица, я так понял, вы то же решили оставить как есть;
 
остальные изменения - см. в прошлом письме.

ссылка на базу (как договаривались)
https://cloud.mail.ru/public/H5zP/5oUxtH7Pb 

конф. без пароля, на 1С Павел 274275

**Comment by vvveryaskin** [comment]
бля, это 7((((
Ну так то да, вроде исправили то, что было.
От меня еще вопрос: не увидел мастера в заказ наряде, нет имэйла сотрудников

**Comment by Shtukin** [comment]
да, это 7(, на стороне дилера программист добавит необходимые поля.

**Comment by Shtukin** [comment]
выставлен счет, дилер будет оплачивать

**Comment by Shtukin** [comment]
Доступ в прод
api@dealer61.ru
i0fxuctc

**Comment by Shtukin** [comment]
ответственный за внедрение  Донсков Дмитрий  заместитель директора моб.:+7(917)841-54-77 donskov@domograf.com

**Comment by Shtukin** [comment]
18.04 передана конфигурация дилеру.

**Comment by vvveryaskin** [comment]
На серваке у них  операционка WIN2003S SE SP2
Не поддерживаются наши запросы. Не знаю как они будут выходить из положения
